eng/pipelines/extra-platforms/runtime-extra-platforms-maccatalyst.yml

```diff
@@ -75,3 +75,44 @@ jobs:
         creator: dotnet-bot
         interpreter: true
         testRunNamePrefixSuffix: Mono_$(_BuildConfig)
+
+# #
+# # Build the whole product using Native AOT for maccatalyst and run runtime tests
+# #
```

Comment:
Are we planning to have this as part of this PR?
If not, it should be removed, or have an appropriate comment